response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -440,11 +439,6 @@ def get_collection_exercises_with_samples_by_survey_id(survey_id):
 
     ce_list = get_collection_exercises_by_survey(survey_id)
 
-    for ce in ce_list:
-        sample_summary_id = get_linked_sample_summary_id(ce["id"])
-        if sample_summary_id:
-            ce["sample_summary"] = get_sample_summary(sample_summary_id)
-
```

Comment:
Double check for other calls to get_linked_sample_summary_id and get_sample_summary. There are calls present in build_collection_exercise_details() in collection_exercise.py are they still required ?

Reply:
I have tried removing the other usage of that function, but it results in a 500 error and several test failures. It doesn't seem like something that can be removed without significantly refactoring the code.

Reply:
Looking into it I don't think we can remove any of the other calls to sample as they collect the ingest date time rather than the sample size and links